lua/roslyn/lsp.lua

```diff
@@ -82,6 +86,50 @@ function M.start(bufnr, root_dir, on_init)
         end
     end
 
+    config.on_attach = function(client, attach_bufnr)
+        local original_request = client.request
+        local last_diagnostic = nil
+        local default_handler = roslyn_config.config.handlers["textDocument/diagnostic"]
+            or vim.lsp.handlers["textDocument/diagnostic"]
+
+        if vim.fn.has("nvim-0.11") == 1 then
```

Comment:
Doesn't this also work with nvim-0.10?😔 I was under the impression that it should work when I looked through the PR for deprecation of `client.request` in favor of `client:request`... Anywas, it is fine if it doesn't work, but it's just sad...

Reply:
Okay yeah I just re-read the comments in the PR now...